spec/requests/api/moves_controller_index_spec.rb

```diff
@@ -208,38 +208,6 @@
           end
         end
       end
-
-      context 'when the move includes both a profile id and a person id' do
-        let(:person) { create(:person) }
-        let!(:move) { create(:move, person_id: person.id, profile_id: person.latest_profile.id) }
-
-        it 'returns the correct person' do
-          get_moves
-          person_id = response_json['data'][0].dig('relationships', 'person', 'data', 'id')
-          expect(person_id).to eq(person.id)
-        end
-      end
-
-      context 'when the move includes neither profile id or person id' do
-        let!(:move) { create(:move, person_id: nil, profile_id: nil) }
-
-        it 'returns the correct person' do
-          get_moves
-          person_id = response_json['data'][0].dig('relationships', 'person', 'data', 'id')
-          expect(person_id).to be_nil
-        end
-      end
-
-      context 'when the move includes only a profile id' do
-        let(:person) { create(:person) }
-        let!(:move) { create(:move, person_id: nil, profile_id: person.latest_profile.id) }
-
-        it 'returns the correct person' do
-          get_moves
-          person_id = response_json['data'][0].dig('relationships', 'person', 'data', 'id')
-          expect(person_id).to eq(person.id)
-        end
-      end
```

Comment:
Do we still have coverage for the serializer `person` method both with and without a profile?

Reply:
Move can't have a person_id anymore

Reply:
We are validating that a person can be surfaced for a move, yeah.